Makefile

```diff
@@ -273,12 +275,27 @@ schemas_test:
 	(cd schemas && poetry run pytest)
 
 schemas_check: schemas_install schemas_black schemas_ruff schemas_test
+	(cd schemas && poetry run pydantic2ts --module mozilla_nimbus_schemas.jetstream --output ./test_index.d.ts --json2ts-cmd "yarn json2ts")
+	diff schemas/test_index.d.ts schemas/index.d.ts || (echo nimbus-schemas typescript package is out of sync please run make schemas_build;rm schemas/test_index.d.ts;exit 1)
+	rm schemas/test_index.d.ts && echo "Done. No problems with schemas."
```

Comment:
Oh right we're running these in the host env.  We should put all this in a container at some point, prolly easier for people.  Also instead of doing this write file then rm thing, maybe better to just write to `/tmp`.  Trying to rm a file like this always gets spotty.  In the other ones we write and then forget about it because when the container exits the file goes with it.

Reply:
Yea, I was thinking of this and I like the `/tmp` folder option as a temporary (pun intended) solution. Added that for now and I'll create a ticket to migrate this to docker.